src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14429,12 +14440,19 @@ void VRFCondenserEquipment::VRFOU_CalcCompC(EnergyPlusData &state,
                     goto Label13;
                 }
 
-                if (CapDiff > (Tolerance * Cap_Eva0)) NumIteCcap = 999;
+                if (CapDiff > (Tolerance * Cap_Eva0)) {
```

Comment:
cycling ratio calculation is moved to here

Reply:
This logic does not seem right to me. At line 14439 this same conditional is used. It says if the capacity difference is greater than some tolerance keep iterating. So when we get to line 14447, it's either due to max iterations or the capacity difference is less than tolerance. I guess if capacity difference is less than tolerance then we converged on the capacity that requires the compressor to run the entire time step. But if `(NumIteCcap >= 30)` then what does that mean? I'm not sure it means the compressor is cycling. I think it might mean the compressor is cycling if `Cap_Eva1 - Cap_Eva0` is positive but you don't know that since `CapDiff = std::abs(Cap_Eva1 - Cap_Eva0);` I would think that should be accounted for here. I'm not sure though since this is a very detailed model.

if (CapDiff > (Tolerance * Cap_Eva0) && (Cap_Eva1 - Cap_Eva0) >= 0.0)

Reply:
Indeed. If Cap_Eva1 - Cap_Eva0 < 0 then it should not cycle.